tests/lib/Template/JSCombinerTest.php

```diff
@@ -87,14 +83,10 @@ public function testProcessUncachedFileNoAppDataFolder(): void {
 		$this->config
 			->expects($this->exactly(2))
 			->method('getValue')
-			->withConsecutive(
-				['debug'],
-				['installed']
-			)
-			->willReturnOnConsecutiveCalls(
-				false,
-				true
-			);
+			->willReturnMap([
+				['debug', '', false],
```

Comment:
```suggestion
				['debug', $this->anything(), false],
```

Is that sort of thing working with `willReturnMap`?
I’m a bit frightened by all these tests being tied to the default value passed.

Reply:
No, unfortunately it does not.